src/PlutotvData.cpp

```diff
@@ -476,6 +476,25 @@ PVR_ERROR PlutotvData::GetEPGForChannel(int channelUid,
           {
             tag.SetIconPath(epgData["episode"]["thumbnail"]["path"].GetString());
           }
+
+          // series title / episode name
+          if (epgData["episode"].HasMember("series") &&
+              epgData["episode"]["series"].HasMember("name") &&
+	      epgData["episode"]["series"]["name"].IsString() &&
+	      epgData["episode"].HasMember("name") &&
+	      epgData["episode"]["name"].IsString())
+           {
+              // series title
+              tag.SetTitle(epgData["episode"]["series"]["name"].GetString());
```

Comment:
Index access is quite an expensive operation and you can have thousands of epf tags. Wouldn't it make sense to introduce variables instead of accessing the array via index multiple tomes?